src/Native/Runtime/unix/unixasmmacrosarm64.inc

```diff
@@ -88,6 +88,14 @@ C_FUNC(\Name):
         stp \reg1, \reg2, [sp, \ofs]!
         .ifc \reg1, fp
         mov fp, sp
+
+        .cfi_def_cfa 29, 16
```

Comment:
CoreCLR has `cfi_def_cfa_register fp` here: https://github.com/dotnet/runtime/commit/264c405cbfcefcebb39298fcb8e9c9486a1e92d6#

Would it be a better fix for the issue you are seeing?

Reply:
Using cfi_def_cfa_register instead of cfi_def_cfa requiers positive offsets for the cfi_offset. The debugger (Android) doesn't seem to like this as it still cuts the callstack.

The C++ compiler seem to always uses cfi_def_cfa with a positive offset and than cfi_offset with negativ. If I would not need these two cfi_offset for FP and LR it would not matter but the debugger doesn't work without these.

Reply:
I just noticed the current version of the CLR files has even more changes. Let me try if this variant works with the debugger

Reply:
OK, looks like I can fix it with the code from the CLR file but it will require some more additional work. I will set the PR back to draft until I am finished